addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1013,13 +1013,16 @@ def _set_last_seen_message(self, last_message, allow_older=False):
             'last_seen_dt': fields.Datetime.now(),
         })
 
+    def _get_allowed_channel_types_to_mark_as_fetched(self):
+        return ['chat']
+
     def channel_fetched(self):
         """ Broadcast the channel_fetched notification to channel members
         """
         for channel in self:
             if not channel.message_ids.ids:
                 return
-            if channel.channel_type != 'chat':
+            if channel.channel_type not in self._get_allowed_channel_types_to_mark_as_fetched():
```

Comment:
@seb-odoo not sure the purpose of that code

Reply:
Show small "v" when correspondent received the message